examples/complete/main.tf

```
@@ -106,10 +106,9 @@ module "vpc_endpoints" {
     s3 = {
       service             = "s3"
       private_dns_enabled = true
-      dns_options = {
-        private_dns_only_for_inbound_resolver_endpoint = false
-      }
-      tags = { Name = "s3-vpc-endpoint" }
+      type                = "Gateway"
```

I found this PR when trying to figure out how to set an S3 gateway endpoint, but I think the option key here is `service_type`, not `type`:
```suggestion
      service_type        = "Gateway"
```